grails-app/assets/javascripts/rollbar-config.js

```diff
@@ -0,0 +1,10 @@
+// Configuration for rollbar error reporting
+
+var _rollbarConfig = {
+    accessToken: '',  //TODO - get this from somewhere that's not in (public)repository
```

Comment:
maybe pep8 style inline-commenting

Reply:
Indentional because I need to change these values for things to work after which I'll delete the comments

Reply:
still a crime